config/sync/core.extension.yml

```diff
@@ -39,6 +40,7 @@ module:
   hoeringsportal_content_blocks: 0
   hoeringsportal_data: 0
   hoeringsportal_deskpro: 0
+  hoeringsportal_base_fixtures: 0
```

Comment:
The fixtures module should only be installed on demand, e.g. during development and testing, so it should not be added in `core.extensions.yml` (which will install it in production as well).

Reply:
Removed